mautic/mautic
# [UX] Change “Clear cache” button label and tooltip on Marketplace

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [X]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #13328  



#### Description:

The “Clear cache” button label and tooltip on the Marketplace are currently dev-oriented, which may not be easily understood by non-technical users. This issue can be resolved by rewording the label and tooltip to use more user-friendly language that is clear and understandable to all users, regardless of their technical background.

#### Impact
Addressing this issue would improve the user experience for a broader range of users, particularly those without a technical background. It would make the platform more accessible and intuitive, potentially increasing user engagement and satisfaction. This could also reduce the number of support queries related to this feature, saving time for both marketers and companies support staff.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open marketplace
3. Look at the button
4. Hover the mouse over the button and read the tooltip

![image](https://github.com/mautic/mautic/assets/116097999/c5f5d0ea-3fe6-46da-ba88-abb9fe2aa4f3)
